### PR TITLE
Feature/tabbar

### PR DESCRIPTION
**Pull request by ethanswift**
**What this PR does**:
Tab bar controller view: 
Screen Shot 2020-05-19 at 6 37 53 PM
Screen Shot 2020-05-19 at 6 38 02 PM
Screen Shot 2020-05-19 at 6 38 15 PM

### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #

recreates the current tab bar view with the one compatible with design 

**Special notes for reviewers**:

**Additional comments**:

### CODE REVIEW

File: FightPandemics/Features/TabBar/RootTabBarController.swift [hunk 1]
```diff
@@ -143,4 +181,33 @@ extension RootTabBarController: UITabBarControllerDelegate {
...
+        } else if tabBar.items?.firstIndex(of: item) == 2 {
+            addend = 18
+        } else if tabBar.items?.firstIndex(of: item) == 3 {
+            addend = 12
+        }
+        let dotView = UIView()
+        if tabBar.viewWithTag(100) != nil {
+            tabBar.viewWithTag(100)?.removeFromSuperview()
+        }
+        dotView.backgroundColor = .blue
```
Comment (stavares843) [inline FightPandemics/Features/TabBar/RootTabBarController.swift]: doesnt seem the same as figma

figma is 425AF2

https://jmp.sh/nRPYj5k

**Review by stavares843 — COMMENTED** [review]
tapping on profile tab the dot appears above the copy

https://jmp.sh/wZ7bqTm

File: FightPandemics/Features/TabBar/RootTabBarController.swift (hunk 1 above)
Comment (ethanswift) [reply FightPandemics/Features/TabBar/RootTabBarController.swift]: right!!

**Comment by ethanswift** [comment]
> tapping on profile tab the dot appears above the copy
> 
> https://jmp.sh/wZ7bqTm

this is because of the code itself, it has to be regulated.

**Review by stavares843 — COMMENTED** [review]
on the bottom has this effect

https://jmp.sh/ex7j3RD

**Comment by haidalinda** [comment]
Screen Shot 2020-05-19 at 5 29 12 PM

profile title shows along with dot ^

**Comment by haidalinda** [comment]
> profile title shows along with dot ^

you should remove `RootTabBarController`'s conformance to `UITabBarControllerDelegate` - that was a set up to have the right-most tab slide in a hamburger menu (previous design)

**Review by haidalinda — CHANGES_REQUESTED** [review]
see comments for requested changes

but in general - nice work @ethanswift !

**Comment by stavares843** [comment]
> tapping on profile tab the dot appears above the copy

jmp.sh/wZ7bqTm

> on the bottom has this effect

jmp.sh/ex7j3RD

those still happens

**Comment by ethanswift** [comment]
> > tapping on profile tab the dot appears above the copy
> 
> jmp.sh/wZ7bqTm
> 
> > on the bottom has this effect
> 
> jmp.sh/ex7j3RD
> 
> those still happens

I think it's now fixed, please check again

**Comment by stavares843** [comment]
>tapping on profile tab the dot appears above the copy

> jmp.sh/wZ7bqTm

fixed

> on the bottom has this effect

> jmp.sh/ex7j3RD

still happens

on SE lacks a bit spacing on the bottom also

https://jmp.sh/8WT9HSO

**Review by stavares843 — COMMENTED** [review]
>tapping on profile tab the dot appears above the copy

> jmp.sh/wZ7bqTm

fixed

> on the bottom has this effect

> jmp.sh/ex7j3RD

still happens

on SE lacks a bit spacing on the bottom also

https://jmp.sh/8WT9HSO

**Comment by ethanswift** [comment]
> > tapping on profile tab the dot appears above the copy
> 
> > jmp.sh/wZ7bqTm
> 
> fixed
> 
> > on the bottom has this effect
> 
> > jmp.sh/ex7j3RD
> 
> still happens
> 
> on SE lacks a bit spacing on the bottom also
> 
> https://jmp.sh/8WT9HSO

you sure! this is my image : 
Screen Shot 2020-05-20 at 2 42 07 AM

**Comment by stavares843** [comment]
strange, is fine now

maybe some bug on the simulator

**Review by stavares843 — APPROVED** [review]

**Review by haidalinda — APPROVED** [review]

**Pull request merged**